CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+### 1.10.0 / 2018-01-03
+* Add support for Puppet 5
+  * Note: you need to set 'puppet_collection' to 'puppet5' to test Puppet 5 and
+    'aio' (or leave it out) to test Puppet less than 5
```

Comment:
I have to change more than just PUPPET_VERISON for this to work?

Reply:
@jeefberkey Yes, there are no hooks early enough into Beaker to presently make this work and changes will need to be done as a follow on ticket since they will be more expansive.

Reply:
Is there a spec helper change we can do or something to avoid doing that?

Reply:
My `spec_helper_acceptance.rb` has a section that looks like 
```ruby
unless ENV['BEAKER_provision'] == 'no'
  hosts.each do |host|
    # Install Puppet
    if host.is_pe?
      install_pe
    else
      install_puppet
    end
  end
end
```

I'm thinking we could make another function or something and throw that in a logic block like this